README.md

```diff
@@ -10,19 +10,20 @@ The Interop Test Runner aims to automatically generate an interop matrix by runn
 The Interop Runner is written in Python 3. You'll need to install the
 following softwares to run the interop test:
 
-- Python3 modules. Run the following command:
+* Python3 modules. Run the following command:
 
-```bash
-pip3 install -r requirements.txt
-```
+   ```bash
+   pip3 install -r requirements.txt
+   ```
```

Comment:
This doesn't look correct.

Reply:
It is? You gotta indent the block if it's supposed to be part of the bullet.

Reply:
I see, makes sense.